emulated_hue/api.py

```diff
@@ -436,9 +438,23 @@ async def async_change_config(self, request: web.Request, request_data: dict):
         # just log this request and return succes
         LOGGER.debug("Change config called with params: %s", request_data)
         for key, value in request_data.items():
-            await self.config.async_set_storage_value("bridge_config", key, value)
+            if key == "linkbutton" and value and not self.config.link_mode_enabled:
+                await self.config.async_enable_link_mode()
+            else:
+                await self.config.async_set_storage_value("bridge_config", key, value)
         return send_success_response(request.path, request_data, username)
 
+    async def async_scene_to_full_state(self) -> dict:
```

Comment:
Will this also support scenes where the lights have different settings ?

Reply:
What do you mean by different settings? All this does is delete the lightstates and add the lights in the group. Do you have a better way of implementing this? I'm not very satisfied with how I've implemented it, but it does work.

Reply:
Never mind. I'll give it a test once merged.